apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/R4ExplanationOfBenefitResourceProviderIT.java

```diff
@@ -1442,7 +1632,9 @@ private Beneficiary loadSampleA() {
    * Find the first Beneficiary from a record list returned by {@link ServerTestUtils#loadData(List}
    *
    * @param loadedRecords to use
-   * @return the first Beneficiary
+   * @param loadedRecords the loaded records
+   * @return the first Beneficiary*
+   * @return the beneficiary
```

Comment:
not sure about the 2 'return' entries for JavaDoc.

Reply:
good catch, fixed it.